client/ayon_tvpaint/api/pipeline.py

```diff
@@ -474,6 +472,8 @@ def get_containers():
     output = get_workfile_metadata(SECTION_NAME_CONTAINERS)
     if output:
         for item in output:
+            item["schema"] = "ayon:container-3.0"
+            item["id"] = AYON_CONTAINER_ID
```

Comment:
I understand the reasoning of moving it here - but it's good to be aware that the schema being imprinted on the node itself would be what could be used down the line to identify "which version" of imprinted schema it was, so that if needed it could be 'updated' in the future. By removing it - we can only assume that the item's data is correct no matter how we 'change' the schema in the future.

Not saying this is bad because we've rarely changed it... but it _may_ allow us to be less flexible in the future.

Reply:
I'm not aware or any change in last few years except from "removing" required keys, or changing schema name `avalon` > `openpype` > `ayon`. Now, `schema` and `id` should be changed, but nothing really changed for tvpaint integration, only 2 keys required by ayon-core.

And if that happens, then the missing `schema` and `id` is the difference to find out what schema it was. Technically it could be any schema up to `ayon:container-3.0`, because except `id` and `schema` nothing changed over time...

Reply:
Fine with me. Just wanted to make that point.